test/export/test_torchbind.py

```diff
@@ -351,6 +414,188 @@ def forward(self, arg0_1, attr, arg1_1):
     return (getitem_3, add_1)""",  # noqa: B950
         )
 
+    @parametrize("make_fx_tracing_mode", ["fake", "symbolic"])
+    def test_make_fx_tensor_queue_methods(self, make_fx_tracing_mode):
+        test = self
+
+        class Model(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(3, 2)
+                self.check_tq_is_fake = True
+
+            def forward(self, tq, x):
+                if self.check_tq_is_fake:
+                    test.assertTrue(isinstance(tq, FakeScriptObject))
+                tq.push(x.cos())
+                tq.push(x.sin())
+                x_cos = tq.pop() + tq.size()
+                x_sin = tq.pop() - tq.size()
+                return x_sin, x_cos, tq
+
+        mod = Model()
+        tq = torch.classes._TorchScriptTesting._TensorQueue(
+            torch.empty(
+                0,
+            ).fill_(-1)
+        )
+        tq1 = torch.classes._TorchScriptTesting._TensorQueue(
+            torch.empty(
+                0,
+            ).fill_(-1)
+        )
+        x = torch.ones(2, 3)
+        with torch._higher_order_ops.torchbind.enable_torchbind_tracing():
```

Comment:
Sorry, I forgot why -- why do we need to enable this?

Reply:
It's only useful for tracing with real script object. In our case, we've fakified the script object. I should remove it.